src/basic/log.c

```diff
@@ -959,12 +960,9 @@ int log_struct_internal(
 
                 if (journal_fd >= 0) {
                         char header[LINE_MAX];
-                        struct iovec iovec[17] = {};
+                        struct iovec iovec[17];
```

Comment:
won't this skip init to 0?

Reply:
That's the point of the commit. I'm removing initialization because it's initialized below, up to position `n`, which is initially 0.